plugins/pdr_deterministic_plugin/ufm_sim_web_service/isolation_mgr.py

```diff
@@ -64,6 +75,22 @@ def get_counter(counter_name, row, default=0):
         return default
     return val
 
+def calc_symbol_ber_val(row):
+    error_lane_0 = get_counter(Constants.PHY_RAW_ERROR_LANE0, row, default=None)
+    error_lane_1 = get_counter(Constants.PHY_RAW_ERROR_LANE1, row, default=None)
+    error_lane_2 = get_counter(Constants.PHY_RAW_ERROR_LANE2, row, default=None)
+    error_lane_3 = get_counter(Constants.PHY_RAW_ERROR_LANE3, row, default=None)
+    if error_lane_0 and error_lane_1 and error_lane_2 and error_lane_3:
+        symbol_ber_val = sum([
+            get_counter(Constants.PHY_RAW_ERROR_LANE0, row),
```

Comment:
but you have all this values in couple of rows above ? you already did get_counter?
just sum them

Reply:
fixed